internal/pkg/cli/svc_deploy.go

```diff
@@ -371,10 +380,13 @@ func workloadManifest(in *workloadManifestInput) (manifest.WorkloadManifest, err
 	}
 	envMft, err := mft.ApplyEnv(in.envName)
 	if err != nil {
-		return nil, fmt.Errorf("apply environment %s override: %s", in.envName, err)
+		return nil, fmt.Errorf("apply environment %s override: %w", in.envName, err)
 	}
 	if err := envMft.Validate(); err != nil {
-		return nil, fmt.Errorf("validate manifest against environment %s: %s", in.envName, err)
+		return nil, fmt.Errorf("validate manifest against environment %s: %w", in.envName, err)
```

Comment:
(just bc i just saw a very similar line in another PR)
```suggestion
		return nil, fmt.Errorf("validate manifest against environment %q: %w", in.envName, err)
```